machines/vic/reification/appliance.go

```diff
@@ -0,0 +1,416 @@
+// Copyright 2016 VMware, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package reification
```

Comment:
Well, this structure is separated, based on my understanding for George's design document: https://github.com/hickeng/vic/blob/467/doc/design/vic-machine.md
For the VCH manifest part, I think we'll have a common module, to handle manifest and do more for authority delegation.
As those functions are not there yet, I just tried to separate the command parameters with our internal parameters.
Eventually those files should not be inside of installer module. 
Right now, I assume this is just a start. Did you get any plan for the metadata module?